lib/tasks/skin_tasks.rake

```diff
@@ -158,8 +158,22 @@ namespace :skins do
   end
 
   desc "Cache all site skins"
-  task(:cache_all_site_skins => :environment) do
-    Skin.where(cached: true).each{|skin| skin.cache!}
+  task(cache_all_site_skins: :environment) do
+    skins = Skin.in_chooser + [Skin.default]
```

Comment:
The "default" option in the skin chooser is the unset action, which does this:

https://github.com/otwcode/otwarchive/blob/73ee2a428a0a2ce364fd73133e108251ecbd9121/app/controllers/skins_controller.rb#L140-L145

So we should probably pick the skin ID from the admin setting instead:
```rb
Skin.where(id: AdminSetting.default_skin_id).or(Skin.in_chooser).each do |skin|
```
You'll need to set `default_skin: true` on the describe block to set up the admin setting as well.

Reply:
Isn't `Skin.default` the actual [default skin](https://test.archiveofourown.org/skins/1), though, distinct from the "Default"/unset link in the footer? 

https://github.com/otwcode/otwarchive/blob/73ee2a428a0a2ce364fd73133e108251ecbd9121/app/models/skin.rb#L540-L542

...although I guess `AdminSetting.default_skin_id` could be set to Panda Madness rather than Default, and in that case, we'd want to recache Panda Madness, not Default, so the concern still applies.